src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -195,6 +196,12 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
     };
     this.orderStatusInfo = this.getOrderStatusInfo(this.currentOrderStatus);
     this.isStatus = this.generalInfo.orderStatus === OrderStatus.CANCELED;
+    const paymentBonusAccount = this.paymentInfo.paymentInfoDtos.filter(
+      (paymentItem) => paymentItem.receiptLink === PaymentEnrollment.receiptLink
+    );
+    if (paymentBonusAccount) {
```

Comment:
use paymentBonusAccount.length, because an empty array is also true

Reply:
Thank you, code was fixed